### PR TITLE
R4_UK_BRAM_Update_Slot_include

### PR DESCRIPTION
**Pull request by JagatNayak**
Description
----
R4_UK_BRAM_Update_Slot_include

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [x] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

**Comment by JagatNayak** [comment]
image

**Comment by JagatNayak** [comment]
Implementation Note: 
image

**Comment by JagatNayak** [comment]
Example with Include  when **_include=Slot:schedule** :
image

**Comment by JagatNayak** [comment]
Example with Include when _include=Schedule :actor:Practitioner  :
image

**Comment by JagatNayak** [comment]
Patient Authorization Request when _include=Slot:schedule : 

image

**Comment by JagatNayak** [comment]
Patient Authorization Request when _include=Schedule:acto:Practitioner :
image

File: content/millennium/r4/base/workflow/slot.md [hunk 1]
```diff
@@ -91,11 +91,13 @@ Notes:
...
 
-* The `_include` parameter may be provided once with the value `Slot:schedule`. Example: `_include=Slot:schedule`
+* The `_include` parameter may be provided once with the value `Slot:schedule` or `Schedule:actor:Practitioner`. Example: `_include=Slot:schedule` or `_include=Schedule:actor:Practitioner`
 
 * The `_include` parameter may be provided with the `_id` parameter. Example: `_id=21265426-633867-3121665-0&_include=Slot:schedule`
 
-* When `_include` is provided in a request to the closed endpoint, the OAuth2 token must include one of `user/Schedule.read`, `patient/Schedule.read`, or `system/Schedule.read` scopes in addition to the Slot scope.
+* When `_include=Slot:schedule` is provided in a request to the closed endpoint, the OAuth2 token must include one of `user/Schedule.read`, `patient/Schedule.read` or `system/Schedule.read` scopes in addition to the Slot scope.
+
+*  When `_include=Schedule:actor:Practitioner` is provided in a request to the closed endpoint, the OAuth2 token must include  one of `user/Practitioner.read`, `patient/Practitioner.read` or `system/Practitioner.read` scopes in addition to the Slot and Schedule scope.
```
Comment (ashu-bdvt) [inline content/millennium/r4/base/workflow/slot.md:100]:
```suggestion
*  When `_include=Schedule:actor:Practitioner` is provided in a request to the closed endpoint, the OAuth2 token must include  one of `user/Practitioner.read`, `patient/Practitioner.read` or `system/Practitioner.read` scopes and one of `user/Schedule.read`, `patient/Schedule.read` or `system/Schedule.read` scopes in addition to the Slot scope.
```

**Review by ashu-bdvt — CHANGES_REQUESTED** [review]

**Comment by JagatNayak** [comment]
Addressed comment :
image

**Review by NimishaMN — APPROVED** [review]

**Comment by JagatNayak** [comment]
Before Changes :
image

**Comment by JagatNayak** [comment]
Before Changes Implementation :
image

**Review by ashu-bdvt — APPROVED** [review]

**Comment by Surya-Rnsit** [comment]
After Release:
Screenshot 2023-11-10 at 3 25 59 PM
Screenshot 2023-11-10 at 3 24 52 PM

**Pull request merged**